readme.md

```diff
@@ -40,6 +40,7 @@ We humbly suggest the following status codes to be included in the HTTP spec in
     - 737 - FuckThreadsing
     - 738 - Fucking Bundler
     - 739 - Fucking Windows
+    - 740 - Fucking Daylight Saving Time
```

Comment:
74X is already assigned and is classified as meme driven.
Could possibly get away with making it 763.1, I'm not totally sold on that though.